platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -393,16 +411,19 @@ export default class HangingProtocolService extends PubSubService {
    *        the studies to display in viewports.
    * @param protocol is a specific protocol to apply.
    */
-  public run({ studies, displaySets, activeStudy }, protocolId, options = {}) {
+  public run({ studies, displaySets, activeStudy, activeStudyUID }, protocolId, options = {}) {
```

Comment:
No need to provide `activeStudyUID` since we already have `activeStudy`, which includes `activeStudyUID`.